src/ast.hpp

```diff
@@ -110,6 +86,20 @@ namespace Sass {
     void update_pstate(const ParserState& pstate);
     Offset off() { return pstate(); }
     Position pos() { return pstate(); }
+
+    // Some obects are not meant to be compared
+    // ToDo: maybe fallback to pointer comparison?
+    virtual bool operator== (const AST_Node& rhs) const {
```

Comment:
How about leaving it unimplemented instead, so that the error is compile time instead of runtime?

```c++
virtual bool operator== (const AST_Node& rhs) const = 0;
```

Reply:
Because then I need to implement it on every class that extends from this.

Reply:
That seems to indicate that this class shouldn't have this method at all.
OK with doing this later though.

Reply:
I know, but before it was not really better, it actually just ignored those AFAICT. So I consider it at least an improvement.